src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An interface for printable attributes of a person in the address book
```

Comment:
This interface need not be about person/addressbook at all!
References to addressbook or person can be removed here, just what it is supposed to do is enough!